tailwind.config.js

```diff
@@ -7,12 +7,20 @@ export default {
   ],
   theme: {
     extend: {
+      screens: {
+        'sm': '414px',
```

Comment:
I think we shouldn't define custom viewports here. Rather we will go with tailwind viewports. You can remove this.